domain/estimators/recurrent_logistic.py

```diff
@@ -0,0 +1,378 @@
+from abc import ABCMeta, abstractmethod
+import copy
+import logging
+
+import pandas as pd
+import time
+import torch
+from torch.utils.data import TensorDataset, DataLoader
+from tqdm import tqdm
+
+from ..estimator import Estimator
+from utils import dictify_df
+
+
+class RecurrentLogistic(Estimator, torch.nn.Module):
+    """
+    RecurrentLogistic is an Estimator that approximates posterior of
+    p(v_cur | v_init) by training a logistic regression model to predict the current
+    value in a cell given all other initial values using features
+    of the other initial values such as co-occurrence.
+    """
+
+    def __init__(self, dataset, pruned_domain, active_attrs):
+        """
+        :param dataset: (Dataset) original dataset
+        :param pruned_domain: (dict) of tid -> attr -> value for the domain
```

Comment:
I think pruned_domain is a Dict where the key is tid (string)  and value is a Dict where the key is attr(string) and the value is a list of domain values (List[str])